bot/services/message_handling_service.py

```diff
@@ -18,9 +18,23 @@ class MessageHandlingService(BaseService):
     def __init__(self, *, bot):
         super().__init__(bot)
 
-    @BaseService.Listener(Events.on_message_recieved)
-    async def on_message_recieved(self, message: discord.Message) -> None:
-        log.info(f'Message from {message.author}: "{message.content}" Guild {message.guild.id}')
+    @BaseService.Listener(Events.on_message_received)
+    async def on_message_received(self, message: discord.Message) -> None:
+        
+        if isinstance(message.guild, discord.guild.Guild):
+            log.info(f'Message from {message.author}: "{message.content}" Guild {message.guild.id}')
+            log.info(f'{type(message.guild)}')
+        else:
+            embed = discord.Embed(title= f'Bot Direct Message',
+                                    color= Colors.ClemsonOrange,
+                                    description= f'{message.content}')
+            embed.set_footer(text=message.author, icon_url=message.author.avatar_url)
+            log.info(f'Message from {message.author}: "{message.content}" Guild Unknown (DM)')
+            log.info(f'{message.guild}')
+            await self.messenger.publish(Events.on_broadcast_designated_channel, OwnerDesignatedChannels.bot_dm_log, embed)
+            #await message.author.send('👋') # https://discordpy.readthedocs.io/en/latest/faq.html#how-do-i-send-a-dm
+            return
```

Comment:
we dont need this return its implicit at the ned of a function

Reply:
The return is to stop further code execution, not to exit the if-statement. This is why I had to change the `if` to be `if not`. Otherwise, the code will log all normal messages and ignore DMs.